plugins/keyboard-navigation/README.md

```diff
@@ -18,29 +18,29 @@ npm install blockly-plugin-keyboard-navigation --save
 ```
 
 ## Usage
-
-<!--
-  - TODO: Update usage.
-  -->
 ```js
 import * as Blockly from 'blockly';
-import {Plugin} from 'blockly-plugin-keyboard-navigation';
-
+import {Register} from 'blockly-plugin-keyboard-navigation';
```

Comment:
You import `Register` here but do `new NavigationController` below. One of them is probably wrong.

Reply:
ha, yea should be NavigationController. Thanks!